deferrable_gratification.gemspec

```
@@ -24,11 +24,8 @@ Features include:
 
   gem.license = 'MIT'
 
-  gem.required_ruby_version = '>= 1.8.7'
-
   gem.add_dependency 'eventmachine', '>= 1.0.3'
 
-  gem.add_development_dependency "bundler", "~> 1.5"
   gem.add_development_dependency 'rake'
```

I tried running this locally and noticed that we need to restrict `rake` to an older version because of this issue: https://stackoverflow.com/questions/35893584/nomethoderror-undefined-method-last-comment-after-upgrading-to-rake-11

And updating rspec > 2.99 right now breaks specs and it's somewhat out of scope to try to fix that at the moment.

```suggestion
  gem.add_development_dependency 'rake', '< 11.0.1'
```

Also at some point I'd like to set up a GitHub Action to have proper CI running tests here, as I just noticed this is a public repo and so we don't have our internal CI system hooked up to this 😅